code/modules/surgery/organs/organ_internal.dm

```diff
@@ -290,6 +304,8 @@
 	say_mod = "buzzes"
 
 /obj/item/organ/internal/tongue/fly/TongueSpeech(var/message)
+	spawn(0)
+		speakDevilName(message)
```

Comment:
Implement simple event system. Hook it to that. Also helps with organization of  the whole on death effects and on other reactionary effects. I'll understand if you don't want to do that though since it would be a lot of work.